source/wp-content/plugins/wporg-internal-notes/src/notes-list/notes-list-item.scss

```diff
@@ -29,21 +36,22 @@
 	opacity: 0;
 	transition: opacity 300ms ease-in-out;
 
-	.wporg-internal-notes__note:hover & {
+	.wporg-internal-notes__note:hover &,
+	.wporg-internal-notes__note:focus & {
```

Comment:
I'm still seeing an issue with the focus state - I'm focused on the button here:

<img width="273" alt="Screen Shot 2022-03-09 at 3 17 45 PM" src="https://user-images.githubusercontent.com/541093/157529973-97ba1b7f-c175-4578-88d4-3c7cacefc424.png">

I think because either the `:focus` needs to be on the button, or this should use `:focus-within`.

Reply:
Ah good catch. Looks like `:focus-within` does the trick: 93b0f27